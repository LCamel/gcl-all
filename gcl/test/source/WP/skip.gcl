{ True }
skip
{ 0 = 0 }