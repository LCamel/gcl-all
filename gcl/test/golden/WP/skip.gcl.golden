### Result

[ Skip at ./test/source/WP/skip.gcl [9-13] 2:1-5 #299397966446b76f
`True
â‡’
0 = 0`
explanation not available ]

### Source

{ True }
skip
{ 0 = 0 }

### Trace

TraceLog "      [ save ] Parsed"
TraceLog "      [ save ] Converted"
TraceLog "      [ save ] TypeChecked"
TraceLog "      [ save ] Swept"
